### PR TITLE
feat: added no search result found text to event fragment

### PR DESCRIPTION
**Pull request by pranavpandey1998official**
added showNoEventText dunction and called it inside evenetViewModel.events observer when list is empty
added noEventLL to fragment_event

ScreenShot
device-2019-01-10-044252

feat: #857

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt [hunk 1]
```diff
@@ -147,6 +149,11 @@ class EventsFragment : Fragment() {
         return rootView
     }
 
+    private fun showNoEventText(show: Boolean) {
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: `showNoEventText` you are also showing an image. This is a bit confusing

Comment (pranavpandey1998official) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: should I name it to showNoEventPannel or showNoEventLayout

**Comment by RaghavAwasthi** [comment]
@pranavpandey1998official You should squash your commits

**Comment by pranavpandey1998official** [comment]
@iamareebjamal please review it

**Comment by addiegupta** [comment]
> @iamareebjamal please review it

Squash your commits. One PR should contain 1 commit

**Pull request closed without merging**